frappe/public/js/frappe/views/calendar/calendar.js

```diff
@@ -422,6 +422,7 @@ frappe.views.Calendar = class Calendar {
 			name: event[this.field_map.id]
 		};
 
+		args[this.field_map.schedule_date] = me.get_system_datetime(event.start);
```

Comment:
also this is simply ignoring what's specified in `schedule_date` key and deriving it from `start` key. So sending schedule_date in a way isn't doing anything?

Reply:
The framework needs to have a schedule date field since it needs to know what the updated date. since the event object captures updated date in event.start or event.end object I need it pick it from there. From time (start) and To Time (end) is the time for event to start and and end but there is also the need to get capture updated scheduled date if the event date is changed.

Reply:
Got it. It's still better to handle one-off cases in doctype controller. This change will duplicate info of start date in two different fields: start and schedule date.